src/serialPortControl.ts

```diff
@@ -18,21 +18,16 @@ interface ISerialPortDetail {
 export class SerialPortControl {
     public static get serialport(): any {
         if (!SerialPortControl._serialport) {
-            SerialPortControl._serialport = require("../vendor/node-usb-native").SerialPort;
+            SerialPortControl._serialport = require("usb-native").SerialPort;
         }
         return SerialPortControl._serialport;
     }
 
     public static list(): Promise<ISerialPortDetail[]> {
         return new Promise((resolve, reject) => {
-            SerialPortControl.serialport.list(
-                (error: any, ports: ISerialPortDetail[]) => {
-                    if (error) {
-                        reject(error);
-                    } else {
-                        resolve(ports);
-                    }
-                }
+            SerialPortControl.serialport.list().then(
+                ports => resolve(ports),
+                err => reject(err)
```

Comment:
When you open the serial monitor without selecting serial port, the following pop up is shown:
![Annotation 2020-03-19 125636](https://user-images.githubusercontent.com/12337821/77111235-33c17d80-69e4-11ea-9359-6d394da0e261.png)
Do you think you could change the buttons from "Yes" and "No" to "Select port" and "Cancel" or something similar?

Reply:
I can definitely do that, nice catch!